lib/widgets/mixins/twake_context_menu_mixin.dart

```diff
@@ -1,27 +1,31 @@
-import 'package:fluffychat/pages/chat/events/message/message_content_with_timestamp_builder.dart';
+import 'package:fluffychat/widgets/context_menu/context_menu_action.dart';
 import 'package:fluffychat/widgets/context_menu/twake_context_menu.dart';
 import 'package:flutter/material.dart';
 
 /// Show a [TwakeContextMenu] on the given [BuildContext]. For other parameters, see [TwakeContextMenu].
 mixin TwakeContextMenuMixin {
-  void showTwakeContextMenu({
+  Future<int?> showTwakeContextMenu({
+    required List<ContextMenuAction> listActions,
     required Offset offset,
     required BuildContext context,
-    required ContextMenuBuilder builder,
     double? verticalPadding,
     VoidCallback? onClose,
   }) async {
-    await showDialog(
+    int? result;
+    await showDialog<int>(
```

Comment:
can you allow it return `dynamic` type, because in the future, we may need it

Reply:
It can be changed to return `dynamic` but I'm concerned that it might not be clear. IMO, in the future when we need to change or add other types, we'll change it. WDYT?

Reply:
Updated!